bindings/nodejs/lib/types/wallet/account.ts

```diff
@@ -91,11 +91,11 @@ export interface SyncOptions {
     syncPendingTransactions?: boolean;
     /** Specifies what outputs should be synced for the ed25519 addresses from the account. */
     account?: AccountSyncOptions;
-    /** Specifies what outputs should be synced for the address of an alias output. */
-    alias?: AliasSyncOptions;
+    /** Specifies what outputs should be synced for the address of an account output. */
+    account?: AccountSyncOptions;
     /** Specifies what outputs should be synced for the address of an nft output. */
     nft?: NftSyncOptions;
-    /** Specifies if only basic outputs with an AddressUnlockCondition alone should be synced, will overwrite `account`, `alias` and `nft` options. Default: false. */
+    /** Specifies if only basic outputs with an AddressUnlockCondition alone should be synced, will overwrite `account`, `account` and `nft` options. Default: false. */
```

Comment:
hmmmmmm how did this happen

Reply:
I think it's technically correct but we just have this account/account name clash, will have a look

Reply:
Reverted